core/new-gui/src/app/dashboard/component/feature-container/user-project-section/user-project-section.component.ts

```diff
@@ -0,0 +1,129 @@
+import { Component, OnInit } from "@angular/core";
+import { UserProjectService } from "../../../service/user-project/user-project.service";
+import { Project } from "../../../type/project";
+import { Router } from "@angular/router";
+import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
+import { NzMessageService } from "ng-zorro-antd/message";
+
+export const ROUTER_PROJECT_BASE_URL = "/dashboard/project";
+
+@UntilDestroy()
+@Component({
+  selector: "texera-user-project-section",
+  templateUrl: "./user-project-section.component.html",
+  styleUrls: ["./user-project-section.component.scss"]
+})
+export class UserProjectSectionComponent implements OnInit {
+  public projectEntries: Project[] = [];
```

Comment:
please use readonly data structures.

Reply:
will incorporate this together with PR to unify similar logic and components